phpBB/includes/acp/acp_bbcodes.php

```diff
@@ -280,6 +281,7 @@ function main($id, $mode)
 							WHERE bbcode_id = ' . $bbcode_id;
 						$db->sql_query($sql);
 						$cache->destroy('sql', BBCODES_TABLE);
+						$phpbb_container->get('text_formatter.cache')->invalidate();
```

Comment:
Why are you not using your BBCodeMonkey in this class?
AFAIK, it is a superset of phpBB's BBCode definition for custom BBCode...